### PR TITLE
update vocab section with new/old sections

### PR DESCRIPTION
**Pull request by twant**
@letakeane lmk if this is what you had in mind!

### CODE REVIEW

File: lessons/module-1/js-classes.md [hunk 1]
```diff
@@ -6,29 +6,33 @@ tags: js, introduction, constructor functions, this, classes, objects
 
 ## Learning Goals
 
-* Understand what `this` is and how it changes based on context
-* Understand what a class is and what object instances are
+- Understand what `this` is and how it changes based on context
+- Understand what a class is and what object instances are
 
-## Vocabulary
+## New Vocabulary
```
Comment (letakeane) [inline lessons/module-1/js-classes.md:12]: Yes, I love this! Such a great way for students to segment their learning into more digestible bites!

File: lessons/module-1/js-classes.md [hunk 2]
```diff
@@ -6,29 +6,33 @@ tags: js, introduction, constructor functions, this, classes, objects
...
-- `Value` The data referenced by a Key
 - `Property` Another word for a key-value pair on an object
 - `Method` A function on an Object
-- `Dot Notation` Notation to access a Value on an Object, explicitly specifies the Key
-- `Bracket Notation` Notation to access a Value on an Object, usually specifies a Key via a variable
 - `this` A JavaScript keyword with a value that changes depending on the context in which it's used
 - `Class` A constructor that allows us to create multiple instances
 - `Object Instance` Objects that contain the data and functionality defined in the class
 
+## Familiar or Save-for-Later Vocab
```
Comment (letakeane) [inline lessons/module-1/js-classes.md:20]: Does the "save for later" include words that will be addressed later in the lesson once the fundamental new concepts are covered?

**Review by letakeane — APPROVED** [review]
I love this update! We should plan as a team to make this the norm, and update our lessons accordingly!

File: lessons/module-1/js-classes.md (hunk 2 above)
Comment (twant) [reply lessons/module-1/js-classes.md:20]: I'm imagining it as vocab that will be covered more in later lessons. For example, bracket and dot notation has an entire lesson devoted to it later in Mod 1.

**Pull request merged**